aptos-move/aptos-gas/src/move_stdlib.rs

```diff
@@ -4,24 +4,26 @@
 use move_stdlib::natives::GasParameters;
 
 crate::natives::define_gas_parameters_for_natives!(GasParameters, "move_stdlib", [
-    [.bcs.to_bytes.per_byte_serialized, "bcs.to_bytes.per_byte_serialized", 1],
-    [.bcs.to_bytes.failure, "bcs.to_bytes.failure", 10],
+    [.bcs.to_bytes.per_byte_serialized, "bcs.to_bytes.per_byte_serialized", 10],
+    [.bcs.to_bytes.failure, "bcs.to_bytes.failure", 1000],
 
-    [.hash.sha2_256.base, "hash.sha2_256.base", 1],
-    [.hash.sha2_256.per_byte, "hash.sha2_256.per_byte", 1],
-    [.hash.sha3_256.base, "hash.sha3_256.base", 1],
-    [.hash.sha3_256.per_byte, "hash.sha3_256.per_byte", 1],
+    [.hash.sha2_256.base, "hash.sha2_256.base", 3000],
```

Comment:
Same here

Reply:
This is based on benchmarks. There seems to be some "startup costs" associated with these sha functions. Maybe @alinush could explain why?